src/AspNet.Security.OAuth.Fitbit/FitbitAuthenticationHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.Framework.Internal;
+using Newtonsoft.Json.Linq;
+
+namespace AspNet.Security.OAuth.Fitbit {
+    /// <summary>
+    /// Contains static methods that allow to extract user's information from a <see cref="JObject"/>
+    /// instance retrieved from Fitbit after a successful authentication process.
+    /// </summary>
+    public static class FitbitAuthenticationHelper {
+        /// <summary>
+        /// Gets the identifier corresponding to the authenticated user.
+        /// </summary>
+        public static string GetIdentifier([NotNull] JObject user) => user.SelectToken("user.encodedId").ToString();
```

Comment:
Have you considered using `user.Value<JObject>()?.Value<string>("encodedId")`? I think it would be a bit more efficient.